src/destination_io.h

```diff
@@ -13,11 +13,16 @@ typedef boost::variant<CTxDestination, libzcash::SaplingPaymentAddress> CWDestin
 
 namespace Standard {
 
+    struct DecodeOptions {
```

Comment:
see the comment in the rpc about this struct, I would have rather done a function that takes 3 bool

Reply:
Yeah I had that initially and then changed it not married to this method can just add the additional params